custom/icds_reports/templates/icds_reports/icds_app/additional-filter.html

```diff
@@ -16,7 +16,16 @@ <h3 class="modal-title" id="modal-title">Select Additional Filters<i style="floa
             <label for="email">Age:</label>
             <ui-select ng-model="$ctrl.selectedAge" theme="bootstrap" on-select="$ctrl.onSelect($item, $index)">
                 <ui-select-match placeholder="Age"><span ng-bind="$select.selected.name"></span></ui-select-match>
-                  <ui-select-choices repeat="age.id as age in $ctrl.ages| propsFilter: {name: $select.search}">
+                  <ui-select-choices repeat="age.id as age in $ctrl.ages | propsFilter: {name: $select.search}">
+                    <div ng-bind-html="age.name"></div>
+                  </ui-select-choices>
+            </ui-select>
+        </div>
+        <div class="form-group" style="margin: 20px 50px;" ng-show="$ctrl.filters.indexOf('ageServiceDeliveryDashboard') === -1">
+            <label for="email">Age:</label>
```

Comment:
why is this `email`

Reply:
That's not required as `on-select` is used, I am going to remove it from all additional filters.